engine/src/test/java/org/camunda/bpm/engine/test/api/task/TaskServiceTest.java

```diff
@@ -2228,14 +2226,47 @@ public void testDeleteTaskAttachmentWithNullParameters() {
   }
 
   @Test
-  public void testDeleteTaskAttachmentWithTaskIdNull() {
```

Comment:
I changed the name for this test case because to me it looks like the exception thrown here is thrown because the attachment was never created, not because the taskId is null